src/toil/provisioners/gceProvisioner.py

```diff
@@ -34,13 +34,24 @@
 logging.getLogger("urllib3.connectionpool").setLevel(logging.WARNING)
 
 
+# pulled from stack overflow
+# fix for objects getting passed to a json serializer which dies on bytes
+def convert_bytes(data):
```

Comment:
Suggestion, move this function to: https://github.com/DataBiosphere/toil/blob/master/src/toil/lib/compatibility.py